tutorials/R/neonUtilities/neonUtilities.Rmd

```diff
@@ -17,41 +17,46 @@ urlTitle: neonDataStackR
 
 This tutorial goes over how to use the neonUtilities R package 
 (formerly the neonDataStackR package).
-The first three sections, through the `stackByTable()` function, are 
-all you need in order to convert data downloaded from the NEON Data Portal 
-in zipped month-by-site files into aggregated files with all data from the 
-given site(s) and months. The later sections cover additional functions 
-in the package: a conversion function to transform NEON files into 
-GeoCSV format, and wrappers for the API (Application Programming 
-Interface, ; more info on the NEON API 
-<a href="http://data.neonscience.org/data-api" target="_blank">here</a>) that provide alternative 
-pathways for downloading data. 
-Unless noted, the data in this tutorial is NEON single aspirated air 
-temperature.
-
-## Download the Data
-To start, you must have your data of interest downloaded from the 
-<a href="http://data.neonscience.org" target="_blank"> NEON Data Portal</a>.  
 
-The stacking function will only work on zipped Comma Separated Value (.csv) 
-files and not the NEON data stored in other formats (HDF5, etc). 
-
-Your data will download in a single zipped file. 
+The package contains 5 functions:
+
+* `stackByTable()` Takes zip files downloaded from the 
+<a href="data.neonscience.org" target="_blank">Data Portal</a> or 
+<a href="data.neonscience.org/data-api" target="_blank">NEON API</a> 
+(Application Programming Interface), unzips them, and joins 
+the monthly files by data table to create a single file per table.
+* `zipsByProduct()` A wrapper for the NEON API; downloads data 
+based on data product and site criteria. Stores downloaded data 
+in a format that can then be joined by `stackByTable()`.
+* `getPackage()` A wrapper for the NEON API; downloads one 
+site-by-month zip file at a time.
+* `byFileAOP()` A wrapper for the NEON API; downloads remote 
+sensing data based on data product, site, and year criteria. 
+Preserves the file structure of the original data.
+* `transformFileToGeoCSV()` Converts any NEON data file in 
+csv format into a new file with GeoCSV headers.
+
+### If you are only interested in joining data files downloaded from the NEON Data Portal:
```

Comment:
@cklunch This is a great place to use the "Data Tip" CSS styling (in the Style Guide) instead of making all this text an H3 header.